google-cloud-bigtable/acceptance/bigtable/column_family_test.rb

```diff
@@ -28,6 +28,10 @@
     end
   end
 
+  after(:all) do
```

Comment:
Minitest doesn't have `after(:all)`, so this will run after each test in this file is run.

Reply:
Got it.